sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -111,6 +111,7 @@ class CustomerIO internal constructor(
         private var logLevel = CioLogLevel.ERROR
         internal var overrideDiGraph: CustomerIOComponent? = null // set for automated tests
         private var trackingApiUrl: String? = null
+        private var developerMode: Boolean = false
```

Comment:
what do you think about `debugMode` ?  

on another thought, `debugMode` might be a bit confusing with `debug` log mode maybe?

Reply:
I do think a better naming should be done. I would ask the product team to see what they think. 

I think a better name should be chosen because I think this mode should be enabled for every build of a customer's app except for production. Because when they are testing staging or they are doing QA on their mobile app, we should be doing things like saving logs for them. 

Some name that better illustrates that this mode is for helping debugging but can be used by more then just developers developing the app.